polkadot/xcm/docs/src/lib.rs

```diff
@@ -50,6 +50,8 @@
 //!
 //! ## Docs structure
 #![doc = simple_mermaid::mermaid!("../mermaid/structure.mmd")]
+// Frame macros reference features which this crate does not have
+#![allow(unexpected_cfgs)]
```

Comment:
We should create an issue for this

Reply:
You want to eventually add those features to those crates? Because I thought we just want to allow it forever. Cause the features are essentially useless in those crates. I which you could selectively suppress the warnings.

Reply:
The macros should not require these features. For that we have [this](https://github.com/paritytech/polkadot-sdk/blob/9e75647c8cb0286f8d6b48f5e8acdcb386b10235/substrate/primitives/core/src/lib.rs#L425) which enables code, based on features being enabled in the `frame-support` crate. (The correct way to handle this)